spec/services/user_signup_spec.rb

```diff
@@ -0,0 +1,138 @@
+require 'spec_helper'
+
+describe UserSignup do
+
+  describe '#sign_up' do
+    let(:stripe_charge) { double(Stripe::Charge) }
+    let(:service) { UserSignup.new(user) }
+
+    before do
+      allow(StripeWrapper::Charge).to receive(:create).and_return stripe_charge
+      allow(stripe_charge).to receive_messages(succesful?: true, customer_id: "ABC123")
+    end
+
+    context 'with valid input' do
+      let(:user) { Fabricate.build(:user) }
+
+      it 'sets the status to success' do
+        service.sign_up
```

Comment:
When you call sign_up here and specs below, its not clear that you will expect som arguments to be passed in. In reality, you will pass in some token, so it would be good if the specs reflected that need.